core/src/components/textarea/textarea.scss

```diff
@@ -67,13 +67,27 @@
   z-index: $z-index-item-input;
 
   box-sizing: border-box;
+
+  /**
+   * textarea defaults to break-spaces which
+   * can cause extra whitespace to appear
+   * when using newlines before a slotted
+   * label element. To avoid this, we use
+   * "collapse" which is the default value
+   * for an <input />
+   * If textarea is converted to Shadow DOM
+   * then this may not be needed.
+   */
+  white-space-collapse: collapse;
```

Comment:
`<input />` defaults to `collapse` whereas textarea defaults to `preserve` which is why this change was only needed on ion-textarea.